### ISSUE TITLE
--extrawalletargs can't handle options with spaces in them

### ISSUE THREAD

**Issue by jcvernaleo** [decred/decrediton#265]
```
--extrawalletargs='--appdata=/some/crazy/path\ with\ spaces\ in/it'
```

**Comment by oktapodia** [comment]
Fixed, can you close it?

**Comment by jolan** [comment]
FYI, you put "fixes issue #..." it needs to be just "fixes #..." [one of the keywords](https://help.github.com/articles/closing-issues-using-keywords/).

Closed via #729.

**Comment by oktapodia** [comment]
oh! Thanks :D
